2-copy-of-code/lesson-16/data/cart.js

```diff
@@ -1,3 +1,5 @@
+import {validDeliveryOption} from './deliveryOptions.js';
+
 export let cart;

```

Comment:
When I import validDeliveryOption in cart.js all my tests gets deleted and I only left with moneyTests in jasmine. What could be the reason?